pom.xml

```diff
@@ -1701,6 +1701,7 @@
                                     -Xep:FallThrough:ERROR
                                     -Xep:FormatString:ERROR
                                     -Xep:GetClassOnClass:ERROR
+                                    -Xep:HashtableContains:ERROR
```

Comment:
This is a very pointed check to deny usage of a very specific method.
I like the check, but was wondering whether we would want to add it to `.mvn/modernizer/violations.xml` as well.

I don't feel strongly about this, so treat this more as a FYI

Reply:
That's a cool thing, thanks! Probably does not make sense to double an errorprone check there, but now that I know about it... BTW, errorprone has something called Refaster, which can automatically apply transformations to certain patterns, but when I experimented with it some time ago I couldn't find a nice way to integrate with Maven, unfortunately.